core/miner/storage_fsm/precommit_batcher.hpp

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+#include <memory>
+#include <mutex>
+#include "miner/storage_fsm/types.hpp"
+
+namespace fc::mining {
+  using api::SectorPreCommitInfo;
+  using primitives::TokenAmount;
+  using fc::mining::types::SectorInfo;
+
+  class PreCommitBatcher {
+   public:
+    virtual outcome::result<void> addPreCommit(SectorInfo secInf,
+        TokenAmount deposit,
+        SectorPreCommitInfo pcInfo) = 0;
```

Comment:
```suggestion
        const TokenAmount &deposit,
        const SectorPreCommitInfo &pcInfo) = 0;
```

Reply:
addPrecommit batch uses std::move, which is making same functionality like(if we will push const &  to vector it will be copied, if we make copy and move it to vector -> it is the same)

Reply:
fixed